### PR TITLE
Server setup

### PR DESCRIPTION
**Pull request by offiongbassey**
Sever Setup Completed and Database Connected Successfully.

### CODE REVIEW

File: server.js
Comment (eleezs) [inline server.js:1]: change the file name to index.js

File: server/config/Database.js
Comment (eleezs) [inline server/config/Database.js:1]: read this: https://sequelize.org/docs/v6/other-topics/migrations/
I hope it helps with the server file structure.

If you don't understand please ask me.

File: server.js
Comment (offiongbassey) [reply server.js:1]: Okay. I will do that now.

File: server/config/Database.js
Comment (offiongbassey) [reply server/config/Database.js:1]: Thanks for the info and detailed guide. Am done Boss.

Comment (offiongbassey) [reply server/config/Database.js:1]: Thanks for the info and detailed guide. Am done Boss.

File: src/index.js
Comment (eleezs) [inline src/index.js:1]: Don't limit the port the frontend will use. On your cors.

File: src/server/config/config.js
Comment (eleezs) [inline src/server/config/config.js:1]: make the localhost an env variable

File: yarn-error.log
Comment (eleezs) [inline yarn-error.log:1]: add this to git ignore file

File: src/index.js
Comment (offiongbassey) [reply src/index.js:1]: Okay. I have done that.

File: src/server/config/config.js
Comment (offiongbassey) [reply src/server/config/config.js:1]: Noted. Done

File: yarn-error.log
Comment (offiongbassey) [reply yarn-error.log:1]: I have added the yarn.error.log to git ignore file.

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import dotenv from "dotenv";
+import Models from "./server/models";
+
+dotenv.config();
+
+const app = express();
+app.use(cookieParser());
+app.use(express.urlencoded({extended: true}));
+
+app.use(cors("*"));
```
Comment (eleezs) [inline src/index.js]: this is not correct.
remove the argument you passing in the cors

File: README.md
Comment (eleezs) [inline README.md:1]: What about this one, did you read on how to write this.

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import express from "express";
+import cookieParser from "cookie-parser";
```
Comment (eleezs) [inline src/index.js]: this is not been used, please remove it.

File: src/index.js (hunk 1 above)
Comment (offiongbassey) [reply src/index.js]: Done Sir

File: README.md
Comment (offiongbassey) [reply README.md:1]: Yes I have read on it and I've modified it.

File: src/index.js (hunk 2 above)
Comment (offiongbassey) [reply src/index.js]: I have removed it.

**Comment by eleezs** [comment]
Close this Pr and raise another one

**Pull request closed without merging**